samples/Samples.Server/Startup.cs

```diff
@@ -62,7 +61,6 @@ public void Configure(IApplicationBuilder app)
 
         app.UseWebSockets();
 
-        app.UseGraphQLWebSockets<ChatSchema>();
         app.UseGraphQL<GraphQLHttpMiddlewareWithLogs<ChatSchema>>("/graphql", new GraphQLHttpMiddlewareOptions());
```

Comment:
Note that it would be much easier to use ConfigureExecution to add logging rather than to derive from GraphQLHttpMiddleware.

Reply:
I agree if you mean general logging about request execution but middleware provides many virtual methods to override. I propose to override more of them and add logging there to demonstrate the main idea.